src/lib/pixel-select-drop-down/pixel-select-drop-down.tsx

```diff
@@ -199,7 +205,8 @@ export const PixelDropDown = React.forwardRef<HTMLDivElement, DropDownProps>(
       setIsOptionsOpen(false)
       setDropdownPosition({
         ...dropdownPosition,
-        left: '-500px'
+
+        left: ''
```

Comment:
@masabharry you confirmed that the changes in `dropdownPosition` not causing any issues with its display you